mastodon_bot_combined/src/bot3.js

```diff
@@ -0,0 +1,69 @@
+const M = require("./mastodon");
+require("dotenv").config();
```

Comment:
Should all of these repeated requirements be a part of `require("./mastodon");`?